src/components/FolderCreationModal.jsx

```diff
@@ -54,17 +56,29 @@ const FolderCreationModal = ({
     }
   ]
 
-  const baseApiUrl = `${process.env.REACT_APP_BACKEND_URL}/sites/${siteName}${parentFolder ? `/collections/${parentFolder}` : ''}`
 
-  const saveHandler = async () => {
-    // TODO
-    console.log('saving')
-  }
+  const { mutate: saveHandler, isLoading, error: mutateError } = useMutation(
+    () => moveFiles(siteName, [ ...selectedFiles ], slugifyCategory(title), parentFolder),
+    { onSuccess: () => {
+      const redirectUrl = `/sites/${siteName}/folder/${parentFolder ? `${parentFolder}/subfolder/${slugifyCategory(title)}` : slugifyCategory(title)}`
+      setRedirectToPage(redirectUrl)
+      setIsFolderCreationActive(false)
+    }}
+  )
+
+  useEffect(() => {
+    if (mutateError) {
```

Comment:
can use `onError: () => function()` callback from useMutation instead, similar to `onSuccess()`

Reply:
Good point, resolved in https://github.com/isomerpages/isomercms-frontend/pull/369/commits/c479ccafc75503d0676e4e72e1ee24614e0e01f4